.github/workflows/labelize.yml

```diff
@@ -0,0 +1,21 @@
+on: issue_comment
+name: Labelize
+jobs:
+  #----------------------------------------------------------------------------
+  # Labelize
+  #----------------------------------------------------------------------------
+  labelize:
+    name: Labelize
+    runs-on: ubuntu-latest
+    if: ${{ github.event.issue.pull_request }}
+
+    steps:
+      - name: Comment on PR
+        if: contains(github.event.comment.body, '\label ci:')
+        shell: bash
+        run: |
+          label=$(cut -d " " -f2 <<< "${{ github.event.comment.body }}")
+          gh issue edit ${{ github.event.issue.number }} --add-label "$label"
```

Comment:
issue number? Should be the PR identifier, right?

Reply:
should we also have a way to clear labels?

Reply:
> issue number? Should be the PR identifier, right?

It is the issue number, see this:

https://docs.github.com/en/actions/reference/workflows-and-actions/events-that-trigger-workflows#issue_comment-on-issues-only-or-pull-requests-only

> should we also have a way to clear labels?

I'm not entirely needed this is absolutely needed but it can be added.

`\label clear` to remove all labels ?